### PR TITLE
Update seafile.md

### PR DESCRIPTION
**Pull request by Cherryblue**
Updating the guide for Seafile v11+, to mention the differences.

### CODE REVIEW

File: example_configs/seafile.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
+Seafile can be bridged to LLDAP directly, or by using Authelia as an intermediary. This document will guide you through both setups.
+
+## Configuring Seafile (prior to v11.0) to use LLDAP directly
+Seafile's LDAP interface requires a unique, immutable user identifier in the format of `username@domain`. Since LLDAP does not provide an attribute like `userPrincipalName`, the only attribute that somewhat qualifies is therefore `mail`. However, using `mail` as the user identifier results in the issue that Seafile will treat you as an entirely new user if you change your email address through LLDAP. **Note that this isn't true starting Seafile v11.0 and ulterior versions.**
```
Comment (nitnelave) [inline example_configs/seafile.md:5]: Maybe move the note at the front:

Note for seafile before v11: ...

File: example_configs/seafile.md [hunk 2]
```diff
@@ -23,6 +25,34 @@ FILTER = memberOf=cn=seafile_user,ou=groups,dc=example,dc=com
 ```
 * Replace `seafile_user` with the name of your group.
 
+## Configuring Seafile v11.0+ to use LLDAP directly
```
Comment (nitnelave) [inline example_configs/seafile.md:28]: Let's move this section to the top: we want to encourage people to use up to date software (especially in this case)

File: example_configs/seafile.md [hunk 3]
```diff
@@ -23,6 +25,34 @@ FILTER = memberOf=cn=seafile_user,ou=groups,dc=example,dc=com
...
+ENABLE_LDAP = True
+LDAP_SERVER_URL = 'ldap://192.168.1.100:3890'
+LDAP_BASE_DN = 'ou=people,dc=example,dc=com'
+LDAP_ADMIN_DN = 'uid=admin,ou=people,dc=example,dc=com'
+LDAP_ADMIN_PASSWORD = 'CHANGE_ME'
+LDAP_PROVIDER = 'ldap'
+LDAP_LOGIN_ATTR = 'uid'
+LDAP_CONTACT_EMAIL_ATTR = 'mail'
+LDAP_USER_ROLE_ATTR = ''
+LDAP_USER_FIRST_NAME_ATTR = 'cn'
```
Comment (nitnelave) [inline example_configs/seafile.md:44]: That would be givenName, not cn

File: example_configs/seafile.md [hunk 4]
```diff
@@ -23,6 +25,34 @@ FILTER = memberOf=cn=seafile_user,ou=groups,dc=example,dc=com
...
+LDAP_USER_LAST_NAME_ATTR = 'sn'
+LDAP_USER_NAME_REVERSE = False
+```
+
+* Replace `192.168.1.100:3890` with your LLDAP server's ip/hostname and port.
+* Replace every instance of `dc=example,dc=com` with your configured domain.
+
+After restarting the Seafile server, users should be able to log in with their UID and password.
+
+Note : There is currently no ldap binding for users' avatar. [If interested, do mention it to the developers to give more credit to the feature](https://forum.seafile.com/t/feature-request-avatar-picture-from-ldap/3350/6).
```
Comment (nitnelave) [inline example_configs/seafile.md:54]: Nit: make the link only on "mention it" to be clearer?

**Review by nitnelave — CHANGES_REQUESTED** [review]

File: example_configs/seafile.md (hunk 1 above)
Comment (Cherryblue) [reply example_configs/seafile.md]: Sorry for the delay, that's actually my first time playing with pull requests and change requests, so I waited for the right time to do it :).

I've reversed the two sections "priori to v11" and "v11+", and then put the note on the beginning on the section as you advised.

File: example_configs/seafile.md (hunk 2 above)
Comment (Cherryblue) [reply example_configs/seafile.md:4]: You're absolutely right, done.

File: example_configs/seafile.md (hunk 3 above)
Comment (Cherryblue) [reply example_configs/seafile.md]: My mistake, I've changed it.

File: example_configs/seafile.md (hunk 4 above)
Comment (Cherryblue) [reply example_configs/seafile.md]: I have no point of view on this, changed it based on your proposition.

**Review by nitnelave — APPROVED** [review]
Thanks! LGTM

**Pull request merged**